<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emotion recognition</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelDet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>560</width>
      <height>420</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_det_fps">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDet_Text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kimenet</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cap_fps">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bemenet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>500</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mentés</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kép</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="smileProgress">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>340</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Állj</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>531</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Automata</string>
     </attribute>
     <widget class="QTextBrowser" name="chocolate">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Mosolyra kész</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>SVM regresszió</string>
     </attribute>
     <widget class="Plot" name="plotLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>450</width>
        <height>250</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Érzelmek</string>
     </attribute>
     <widget class="EmoSpectrum" name="spectrumLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>450</width>
        <height>250</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Fej iránya</string>
     </attribute>
     <widget class="HeadPoseIndicator" name="headPoseLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>200</width>
        <height>250</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>350</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mosoly</string>
    </property>
   </widget>
   <widget class="QSlider" name="frameCounter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="videoProgress">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>530</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Jelenlegi/Max képkocka</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="blackA">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>480</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Fekete háttér</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="recordA">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>510</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rögzítés</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="contrastSizeA">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>540</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QLabel" name="contrastLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>550</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Kontrasztkiemelés mérete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rewindButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forwardButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>560</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>400</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>380</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mosolyküszöb:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>530</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Kikapcsolva</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gombok</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fej állás + meglepetés</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fej állás + kéz felemelés</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CsoKiválasztó algoritmus:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kiválasztott csoki:</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectedChocolateLabel">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>400</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nézett csoki:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lookedChocolateLabel">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>430</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>460</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mosolyra várunk:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nem</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pontok számozása</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Nincs kiválasztva</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kamera:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>labelDet</zorder>
   <zorder>label_det_fps</zorder>
   <zorder>labelDet_Text</zorder>
   <zorder>label_cap_fps</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_11</zorder>
   <zorder>smileProgress</zorder>
   <zorder>pushButton_14</zorder>
   <zorder>pushButton_13</zorder>
   <zorder>stopButton</zorder>
   <zorder>tabWidget</zorder>
   <zorder>label_2</zorder>
   <zorder>frameCounter</zorder>
   <zorder>videoProgress</zorder>
   <zorder>blackA</zorder>
   <zorder>recordA</zorder>
   <zorder>contrastSizeA</zorder>
   <zorder>contrastLabel</zorder>
   <zorder>rewindButton</zorder>
   <zorder>forwardButton</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_15</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>selectedChocolateLabel</zorder>
   <zorder>label_6</zorder>
   <zorder>lookedChocolateLabel</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>checkBox</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>label_9</zorder>
   <zorder>labelCap</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plot</class>
   <extends>QLabel</extends>
   <header>plot.h</header>
  </customwidget>
  <customwidget>
   <class>EmoSpectrum</class>
   <extends>QLabel</extends>
   <header>emospectrum.h</header>
  </customwidget>
  <customwidget>
   <class>HeadPoseIndicator</class>
   <extends>QLabel</extends>
   <header>headposeindicator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedWriteShape()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_13</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_14</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedPicture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrastSizeA</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>contrastSizeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blackA</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>blackToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frameCounter</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>framePos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clicked_3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>830</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>924</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clicked_2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clicked_A()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clicked_4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clicked_1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>725</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clicked_B()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>798</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clicked_C()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>830</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>878</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clicked_5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>815</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clicked_Kesz()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>830</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>833</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clicked_6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>830</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>883</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clicked_A()</slot>
  <slot>clicked_B()</slot>
  <slot>clicked_C()</slot>
  <slot>clicked_1()</slot>
  <slot>clicked_2()</slot>
  <slot>clicked_3()</slot>
  <slot>clicked_4()</slot>
  <slot>clicked_5()</slot>
  <slot>clicked_6()</slot>
  <slot>clicked_Kesz()</slot>
  <slot>blackToggle(bool)</slot>
  <slot>clickedWriteShape()</slot>
  <slot>clickedCamera()</slot>
  <slot>clickedVideo()</slot>
  <slot>clickedPicture()</slot>
  <slot>clickedHaarIEBM()</slot>
  <slot>contrastSizeChangedA(int)</slot>
  <slot>xShiftChangedA(int)</slot>
  <slot>yShiftChangedA(int)</slot>
  <slot>modelScaleChangedA(double)</slot>
  <slot>modelIterationsChangedA(int)</slot>
  <slot>modelRotChangedA(double)</slot>
  <slot>fitChangedA(bool)</slot>
  <slot>memoFitChangedA(bool)</slot>
  <slot>clickedLoadModelA()</slot>
  <slot>clickedStop()</slot>
  <slot>framePos()</slot>
  <slot>contrastSizeChangedB(int)</slot>
  <slot>xShiftChangedB(int)</slot>
  <slot>yShiftChangedB(int)</slot>
  <slot>modelScaleChangedB(double)</slot>
  <slot>modelRotChangedB(double)</slot>
  <slot>modelIterationsChangedB(int)</slot>
  <slot>fitChangedB(bool)</slot>
  <slot>memoFitChangedB(bool)</slot>
  <slot>clickedLoadModelB()</slot>
  <slot>inUseChangedA(bool)</slot>
  <slot>inUseChangedB(bool)</slot>
  <slot>contrastSizeChanged(int)</slot>
 </slots>
</ui>
